polynote-frontend/polynote/messaging/receiver.ts

```diff
@@ -151,105 +149,112 @@ export class NotebookMessageReceiver extends MessageReceiver<NotebookState> {
                 .catch(err => console.error("Error adding backup", err));
 
             return {
-                ...s,
-                path,
+                path: setValue(path, s.path),
                 cells,
-                cellOrder,
-                config: {...s.config, config: config ?? NotebookConfig.default},
+                cellOrder: setValue(cellOrder),
+                config: { config: config ?? NotebookConfig.default },
                 kernel: {
-                    ...s.kernel,
-                    symbols: [...s.kernel.symbols, ...results.flat()]
+                    symbols: setValue([...s.kernel.symbols, ...results.flat()]) // TODO: does this message ever even happen? Will need to add an op for this if so
```

Comment:
It's referring to receiving a `NotebookCells` message in this way. The `append` operation doesn't allow multiple values, so this use of `setValue` is kind of wonky and could cause problems. I'll update the TODO comment to be clearer about that.

Reply:
Hmm, I see: we don't get cell results from the `Notebook` any more; ever since the incremental loading change we get cell results as `CellResults` messages. AFAICT I don't think we'll ever get cell results in this message 🤔